### ISSUE TITLE
GTK versions conflict

### ISSUE THREAD

**Issue by mstrelex** [dorkbox/SystemTray#17]
Hi,

When I try to use java.awt.Desktop on Xubuntu x64 14.04 the following error pops up:

`Gtk-ERROR **: GTK+ 2.x symbols detected. Using GTK+ 2.x and GTK+ 3 in the same process is not supported`

Any ideas how to avoid this conflict ?

**Comment by kuc** [comment]
Try using
```
SystemTray.COMPATIBILITY_MODE = true;
```
or
```
SystemTray.FORCE_GTK2 = true;
```

See README.md

**Comment by dorkbox** [comment]
I someone wants to contribute a GTK3 version, I'd happily accept a pull request.

**Comment by dorkbox** [comment]
This issue should now be fixed in the 3.0 beta.

3.0 beta is released as a "all-in-one" jar, for a variety of different tray types and configurations. I'm starting a new issue to track the progress for this. I definitely will need help testing in different Linux/Mac/Windows configurations (ie: I have no idea if this works in Windows 10, or the latest MacOsX)

I'm closing this issue and tracking everything in a new one. Please use Issue #22 for all related feedback, problems, etc.
